### PR TITLE
add support for config file

### PR DESCRIPTION
**Pull request by rkusa**
Add support to configure the gRPC server via a config file. This should help users that run multiple servers in parallel and thus need different settings per instance.

See [README](https://github.com/DCS-gRPC/rust-server/blob/config-file/README.md#settings) for instructions of how the config file works.

### CODE REVIEW

File: lua/grpc-mission.lua [hunk 1]
```diff
@@ -1,7 +1,18 @@
--- Set default settings.
 if not _G.GRPC then
   _G.GRPC = {}
 end
+
+-- load settings from `Saved Games/DCS/Config/dcs-grpc.lua`
+do
```
Comment (rurounijones) [inline lua/grpc-mission.lua:6]: Can we add some logging like.

* "Checking for DCS/Config/dcs-grpc.lua file"
* "DCS/Config/dcs-grpc.lua file not found"

To help with debugging?

File: README.md [hunk 2]
```diff
@@ -83,21 +83,36 @@ GRPC.throughputLimit = 600
...
+- above `GRPC.load()` inside of a mission,
+  ```lua
+  GRPC.debug = true
+  GRPC.autostart = true
+  GRPC.load()
+  ```
+
+- inside a `Saved Games\DCS\Config\dcs-grpc.lua` file (useful if run multiple servers in parallel), or
+
+  `Saved Games\DCS\Config\dcs-grpc.lua`:
```
Comment (rkusa) [inline README.md:96]: Wdyt about the name?
Other naming options would be: `grpc.lua`, `DCS-gRPC.lua`

File: lua/grpc-mission.lua (hunk 1 above)
Comment (rkusa) [reply lua/grpc-mission.lua:6]: Done in 8cbf23ab6df327329a3a057aa87d5b61a5b44216 👍

File: README.md (hunk 2 above)
Comment (rurounijones) [reply README.md:96]: Name is fine I think. No strong opinions on alternatives.

**Review by rurounijones — APPROVED** [review]

**Pull request merged**